aws-postgresql.yml

```diff
@@ -240,44 +240,53 @@ provision:
     type: string
     details: The preferred maintenance end hour
     default: "00"
-    enum:
-      "00": 12 am
-      "01": 1 am
-      "02": 2 am
-      "03": 3 am
-      "04": 4 am
-      "05": 5 am
-      "06": 6 am
-      "07": 7 am
-      "08": 8 am
-      "09": 9 am
-      "10": 10 am
-      "11": 11 am
-      "12": 12 pm
-      "13": 1 pm
-      "14": 2 pm
-      "15": 3 pm
-      "16": 4 pm
-      "17": 5 pm
-      "18": 6 pm
-      "19": 7 pm
-      "20": 8 pm
-      "21": 9 pm
-      "22": 10 pm
-      "23": 11 pm
+    enum: *hour_enum
   - field_name: maintenance_end_min
     type: string
     details: The preferred maintenance end minute
     default: "00"
-    enum:
-      "00": Top of the hour
-      "15": 15 minutes
-      "30": 30 minutes
-      "45": 45 minutes
+    enum: *minute_enum
   - field_name: deletion_protection
     type: boolean
     details: Whether deletion protection is enabled. The database cannot be deleted when this value is set.
     default: false
+  - field_name: backup_day
+    type: string
+    details: The preferred automated backup day
+    default: Sun
+    enum: *weekday_enum
+  - field_name: backup_retention_period
+    type: number
+    details: The number of days (1-35) for which automatic backups are kept
+    default: 7
```

Comment:
nitpick: It would be nice to add this information:

> Setting the backup retention period to 0 disables automated backups

> An outage occurs if you change the backup retention period from 0 to a nonzero value or from a nonzero value to 0. This applies to both Single-AZ and Multi-AZ DB instances.